js/renovation/ui/scroll_view/__tests__/scrollable.test.tsx

```diff
@@ -42,17 +42,36 @@ jest.mock('../../../../core/utils/scroll_rtl_behavior', () => () => testBehavior
   describe('Scrollable', () => {
     describe('Render', () => {
       it('should render scrollable content', () => {
-        const scrollView = shallow(viewFunction({ props: { } } as any) as JSX.Element);
-        const scrollViewContent = scrollView.find('.dx-scrollable-wrapper .dx-scrollable-container .dx-scrollable-content');
-        expect(scrollViewContent.exists()).toBe(true);
+        const scrollable = shallow(viewFunction({ props: { } } as any) as JSX.Element);
+        const scrollableContent = scrollable.find('.dx-scrollable-wrapper .dx-scrollable-container .dx-scrollable-content');
```

Comment:
I think we should check exact DOM tree from root to content instead of 'find'

Reply:
Maybe will be better to check stracture using qunit tests?

Reply:
I mean this:
const scrollableContent = scrollable.find('.dx-scrollable-wrapper > .dx-scrollable-container > .dx-scrollable-content');

Reply:
I see. Fixed in https://github.com/DevExpress/DevExtreme/pull/15548/commits/81c0c59a38bb62e1573599a33b088386786b0eaf